portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Alan's projects</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Projects (2018-2020)</h1>
+      <img class="resizeProjects" src="images/cards.png" alt="Playing cards">
+      <img class="resizeProjects" src="images/film.png" alt="Film Trailer">
+      <img class="resizeProjects" src="images/gba.png" alt="GBA game">
+      <img class="resizeProjects" src="images/edutech.png" alt="Educational Game">
+      <img class="resizeProjects" src="images/ui.png" alt="User Interface">
+      <img class="resizeProjects" src="images/space-trader.png" alt="Game built in Java">
```

Comment:
Did you know that you can add a link to images as well? So you can click on the picture and take you to a github repo, website, etc. if you wanted to do any of that! If not, that's perfectly fine too I know this is just practice